### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DhruvaSambrani**
...to include information about inbuilt package manager

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,7 +75,9 @@ Don't worry if it doesn't work immediately! It can take a while for the web page
 
 ### Julia Packages
 
-When your notebook runs on github, no packages are installed. To solve this, you need to **declare a package environment** inside the notebook, using `Pkg`.
+The future is now! With Pluto's new in-built package manager, required packages are stored in the notebook itself!
```
Comment (pankgeorg) [inline README.md:78]: Heyy!!
Can you specify which Pluto version includes that? v0.15.1

**Review by greimel — COMMENTED** [review]
One could also remove the whole section and/or refer to the [wiki](https://github.com/fonsp/Pluto.jl/wiki/🎁-Package-management)

**Comment by fonsp** [comment]
awesome!

**Comment by fonsp** [comment]
Let's do as greimel suggested above. Can you make those changes?

**Pull request merged**